src/MissionManager/SimpleMissionItem.h

```diff
@@ -37,6 +37,8 @@ class SimpleMissionItem : public VisualMissionItem
     Q_PROPERTY(QGroundControlQmlGlobal::AltitudeMode altitudeMode READ altitudeMode WRITE setAltitudeMode       NOTIFY altitudeModeChanged)
     Q_PROPERTY(Fact*            amslAltAboveTerrain     READ amslAltAboveTerrain                                CONSTANT)                           ///< Actual AMSL altitude for item if altitudeMode == AltitudeAboveTerrain
     Q_PROPERTY(int              command                 READ command                WRITE setCommand            NOTIFY commandChanged)
+    Q_PROPERTY(bool             isLoiterItem            READ isLoiterItem                                       NOTIFY isLoiterItemChanged)
```

Comment:
I'd like to see more of the business logic for how this work encapsulated within SimpleMissionItem instead of spreading across both the C++ and Qml code. So instead have a `bool showLoiterRadius` property which deals with all the specifics internally.